Tests/XcodeGenKitTests/ProjectSpecTests.swift

```diff
@@ -207,7 +207,7 @@ class ProjectSpecTests: XCTestCase {
                 var project = baseProject
                 project.schemes = [Scheme(
                     name: "scheme1",
-                    build: .init(targets: [.init(target: "invalidTarget")]),
+                    build: .init(targets: [.init(target: .init(name: "invalidTarget", location: .local))]),
```

Comment:
Let's make `TargetReference` conform to StringLiteralInitializable or whatever it's called. Then all these string inits can stay the same